recipe/build-tiledbsoma-py.sh

```diff
@@ -4,4 +4,5 @@ set -ex
 
 cd apis/python
 
+echo "1.1.1" >> RELEASE-VERSION
```

Comment:
https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/9/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR2

Yeah I agree that it would be better to not hardcode this. I wanted to utilize version from the `meta.yaml` file but couldn't figure out a way to do it.